.dockerignore

```diff
@@ -16,9 +16,8 @@ docker-compose.yml
 **/*.log
 **/dummy
 
-# Other packages
-packages/docs
-packages/sdk-**
+# Docs
+./docs
```

Comment:
The documentation package is here. This will ignore all changes made to the documentation. I don't see in the diff that the docs have been moved to `packages/docs` (that's the old docs package, right?). `./docs` is the new docs package.

Reply:
Sorry, I misunderstood this file as the `.gitignore` not `.dockerignore`. Please disregard.